src/ray/raylet/local_object_manager.cc

```diff
@@ -155,7 +156,13 @@ bool LocalObjectManager::SpillObjectsOfSize(int64_t num_bytes_to_spill) {
     SpillObjectsInternal(objects_to_spill, [this, bytes_to_spill, objects_to_spill,
                                             start_time](const Status &status) {
       if (!status.ok()) {
-        RAY_LOG(ERROR) << "Error spilling objects " << status.ToString();
+        if (status.IsObjectNotFound()) {
```

Comment:
Hmm.. I am not sure if this status is ever returned. Can we just remove it? I have a task to handle this case, and I will update it accordingly.